.github/workflows/ci.yml

```diff
@@ -296,6 +296,11 @@ jobs:
           - os: macos-latest
             platform: darwin-arm
             target: aarch64-apple-darwin
+          - os: ubuntu-latest
```

Comment:
Maybe this CI change should be in a separate PR? Seems unrelated to GQL changes and should be tested out separately.

Reply:
For example, does the default `mingw-w64` image come with protoc and rocksdb installed? Have you tried running the cross commands locally with this windows setup?

Reply:
Sorry, must've slipped in when toying around with cross-compilation yesterday. I had change branches on my terminal but VScode was still open so I think it must've picked up the changes on the wrong branch